modules/announcements/classes/utils.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Elementor\Modules\Announcements\Classes;
+
+use Elementor\Modules\Announcements\Triggers\IsFlexContainerInactive;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Utils {
+	/**
+	 * get_trigger_object
+	 *
+	 * @param $trigger
+	 *
+	 * @return IsFlexContainerInactive|false
+	 */
+	public static function get_trigger_object( $trigger ) {
+		//@TODO - replace with trigger manager
+		switch ( $trigger['action'] ) {
+			case 'isFlexContainerInactive':
```

Comment:
Oh wow, I really don't like the hardcoded value here. Can you please explain why do it this way?